pm_common/pmutil.c

```diff
@@ -8,7 +8,7 @@
 #include "pmutil.h"
 #include "pminternal.h"
 
-#ifdef WIN32
+#if defined(WIN32) | defined(ANDROID)
```

Comment:
This is C code, not CMake.
```suggestion
#if defined(WIN32) || defined(__ANDROID__)
```

Reply:
https://android.googlesource.com/platform/bionic/+/refs/heads/main/docs/defines.md

Reply:
I read that `ANDROID` is "rarely useful" (and why).
And the `|` is also rarely useful.